lib/models/s3-event.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+const crypto = require('crypto');
+
+const S3Event = function (eventData, reqParams) {
+  const {reqHeaders,sourceIp} = reqParams;
+  let eventName = '';
+  const s3Object = {
+    "key": eventData.S3Item.key,
+    "sequencer": ('00' + (Math.floor(Date.now())).toString(16)).toUpperCase()
```

Comment:
According to the [notifications docs](https://docs.aws.amazon.com/AmazonS3/latest/dev/notification-content-structure.html): 

> The sequencers can be of different lengths. So to compare these values, you first right pad the shorter value with zeros and then do lexicographical comparison.

Since there's no guarantee of the sequencer length, you could just use `Date.now().toString(16).toUpperCase()` without prepending `00` and it would match the specified behaviour.